### PR TITLE
Bug 1236490 - Run TV in the system domain r=rexboy

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cc42a1fb3dcee7d6c155b23ffb844f663d99f1d8

File: apps/system/build/build.js [hunk 1]
```diff
@@ -131,6 +131,7 @@ SystemAppBuilder.prototype.inlineDeviceType = function(options) {
 SystemAppBuilder.prototype.enableFirefoxSync = function(options) {
   var fileList = {
     process: [
+      ['index.html'],
```
Comment (albertopq) [inline apps/system/build/build.js]: This was the reason of the Fxa issue

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=93eba61001e4d0c4850c87d346fdd375896be36a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b77ce090039b2f25ff91ff3e94e389af935b442f

**Pull request merged**